core/include/kernel/boot.h

```diff
@@ -69,8 +69,17 @@ void plat_cpu_reset_early(void);
 void plat_primary_init_early(void);
 unsigned long plat_get_aslr_seed(void);
 unsigned long plat_get_freq(void);
-#if defined(_CFG_CORE_STACK_PROTECTOR)
-uintptr_t plat_get_random_stack_canary(void);
+#if defined(_CFG_CORE_STACK_PROTECTOR) || defined(CFG_WITH_STACK_CANARIES)
+/*
+ * plat_get_random_stack_canaries() - get a specified number (ncan) of stack
+ * canaries, each canary has size in bytes.
```

Comment:
`ncan` and `size` are already described below. I think you can simplify with:
` * plat_get_random_stack_canaries() - Get random values for stack canaries`